src/pages/Home/Home.jsx

```diff
@@ -1,17 +1,26 @@
-import React from 'react';
-import './Home.scss';
-import PokeList from '../../PokeList/PokeList';
-import Welcome from '../../Welcome/Welcome';
-import { Pokeview } from '../../Pokeview/Pokeview';
+import React, { useContext } from "react";
+import "./Home.scss";
+import PokeList from "../../PokeList/PokeList";
+import Welcome from "../../Welcome/Welcome";
+import Spinner from "../../Spinner/Spinner";
+import useFetchPokemon from "../../PokeList/useFetchPokemon";
+import { PokemonContext } from "../../context/PokemonContext";
+import { Pokeview } from "../../Pokeview/Pokeview";
 
 const Home = () => {
+  const {
+    state: { isFetching, pokemonMap },
+  } = useContext(PokemonContext);
+  const pokemonList = Object.values(pokemonMap);
+  useFetchPokemon();
```

Comment:
You dont need to fetch pokemon here, PokeList is in charge of that

Reply:
Fixed